fixtures/chauvet-professional/vesuvio-rgba.json

```
@@ -5,11 +5,11 @@
   "meta": {
     "authors": ["Beerady", "Luke Nelson"],
     "createDate": "2022-10-18",
-    "lastModifyDate": "2022-10-18"
+    "lastModifyDate": "2022-11-20"
```

Since the fixture definition didn't really change, I'd not update the last modify date.


```suggestion
    "lastModifyDate": "2022-10-18"
```